infra/repository/locals.tf

```
@@ -0,0 +1,23 @@
+locals {
+  project = "io-p"
+
+  identity_resource_group_name = "${local.project}-identity-rg"
+
+  repo_secrets = {
+    "ARM_TENANT_ID"       = data.azurerm_client_config.current.tenant_id,
+    "ARM_SUBSCRIPTION_ID" = data.azurerm_subscription.current.subscription_id
+  }
+
+  ci = {
+    secrets = {
+      "ARM_CLIENT_ID" = data.azurerm_user_assigned_identity.identity_prod_ci.client_id
+    }
+  }
+
+  cd = {
+    secrets = {
+      "ARM_CLIENT_ID" = data.azurerm_user_assigned_identity.identity_prod_cd.client_id
+    }
+    reviewers_teams = ["all-internal-users", "engineering-team-cloud-eng"]
```

```suggestion
    reviewers_teams = ["io-platform-green-unit", "engineering-team-cloud-eng"]
```

I think `all-internal-users` should be removed in `eng-github-authorization` repo